src/chapters/angular-lsn1/studio.rst

```diff
@@ -1,62 +1,197 @@
+
+.. figure:: ./figures/example-header-component.png
+   :alt: Screen shot of browser showing address localhost:4200, which has a title of Mission Planning Dashboard, a Mission Name, and a Carrier Rocket.
+
+Crew Component
+^^^^^^^^^^^^^^
+Next you need to make a component to show a list of crew members.
+
+1. Create the component by running ``ng g component crew``.
+2. Set the contents of to be:
+
+.. sourcecode:: html
+
+   <h3>Crew</h3>
+   <ul>
+      <li>Jessica Watkins</li>
+      <li>Raja Chari</li>
+      <li>Jasmin Moghbeli</li>
+   </ul>
 
-Add some CSS to change add different colors, fonts, borders, etc. to your
-dashboard.
+3. Add a reference to the header component in ``mission-planning-dashboard/src/app/app.component.html``.
 
-Add links to information pages about the different countries represented by
-the crew.
+.. sourcecode:: html+ng2
+
+   <app-header></app-header>
+   <app-crew></app-crew>
+
+.. figure:: ./figures/example-crew-component.png
+   :alt: Screen shot of browser showing address localhost:4200, which has a title of Mission Planning Dashboard, a Mission Name,a Carrier Rocket, a Crew header, and a list of crew members in an unordered list.
+
+Equipment Component
+^^^^^^^^^^^^^^^^^^^
+Now you need to create a component to show a list of equipment.
+
+1. Create an equipment component named ``equipment``
+2. In the component display data:
+
+   * An ``<h3>`` that contains "Equipment"
+   * A ``<ul>`` that contains ``<li>`` for: Habitat dome, Drones, Food containers, Oxygen tanks
+
+2. Add the component to ``mission-planning-dashboard/src/app/app.component.html`` using the below HTML. Notice the ``<div>`` surrounding the crew and equipment components.
+
+.. sourcecode:: html+ng2
+   :linenos:
+
+   <app-header></app-header>
+   <div class="box">
+      <app-crew></app-crew>
+      <app-equipment></app-equipment>
+   </div>
+
+3. Add CSS to file ``mission-planning-dashboard/src/app/app.component.css`` that will align the crew and equipment lists. Without this CSS the crew list will appear above the equipment list.
+
+.. sourcecode:: css
+   :linenos:
+
+   .box {
+     display: flex;
+     padding: 10px;
+   }
+
+.. figure:: ./figures/example-equipment-component.png
+   :alt: Screen shot of browser showing address localhost:4200, which has a title of Mission Planning Dashboard, a Mission Name, a Carrier Rocket, a Crew header, a list of crew members, and a list of equipment.
+
+.. note::
+
+   A full explanation of ``display: flex;`` is beyond the scope of this book. For more information
+   see `MDN flex box docs <https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox>`_
+   and `CSS Tricks flex box guide <https://css-tricks.com/snippets/css/a-guide-to-flexbox/>`_.
+
+Experiments Component
+^^^^^^^^^^^^^^^^^^^^^
+1. Finally add an experiments component that contains the below HTML:
+
+.. sourcecode:: html
+
+   <h3>Experiments</h3>
+   <ul>
```

Comment:
Again, I suggest placeholders in the HTML referencing an array in ``experiments.component.ts``.